persistent_chains/mainnet/genesis.json

```diff
@@ -0,0 +1,198 @@
+{
+  "genesis_time": "2021-11-11T14:47:05.733203Z",
+  "chain_id": "cheqd-mainnet-1",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "200000",
+      "max_gas": "2000000",
+      "time_iota_ms": "1000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "1576800",
+      "max_age_duration": "7884000",
+      "max_bytes": "200000"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {}
+  },
+  "app_hash": "",
+  "app_state": {
+    "auth": {
+      "params": {
+        "max_memo_characters": "512",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+      ]
+    },
+    "authz": {
+      "authorization": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+      ],
+      "supply": [],
+      "denom_metadata": []
+    },
+    "capability": {
+      "index": "1",
+      "owners": []
+    },
+    "cheqd": {},
+    "crisis": {
+      "constant_fee": {
+        "denom": "ncheq",
+        "amount": "10000000000000"
+      }
+    },
+    "distribution": {
+      "params": {
+        "community_tax": "0.020000000000000000",
+        "base_proposer_reward": "0.010000000000000000",
+        "bonus_proposer_reward": "0.040000000000000000",
+        "withdraw_addr_enabled": true
+      },
+      "fee_pool": {
+        "community_pool": []
+      },
+      "delegator_withdraw_infos": [],
+      "previous_proposer": "",
+      "outstanding_rewards": [],
+      "validator_accumulated_commissions": [],
+      "validator_historical_rewards": [],
+      "validator_current_rewards": [],
+      "delegator_starting_infos": [],
+      "validator_slash_events": []
+    },
+    "evidence": {
+      "evidence": []
+    },
+    "feegrant": {
+      "allowances": []
+    },
+    "genutil": {
+      "gen_txs": [
+      ]
+    },
+    "gov": {
+      "starting_proposal_id": "1",
+      "deposits": [],
+      "votes": [],
+      "proposals": [],
+      "deposit_params": {
+        "min_deposit": [
+          {
+            "denom": "ncheq",
+            "amount": "8000000000000"
+          }
+        ],
+        "max_deposit_period": "1210000s"
+      },
+      "voting_params": {
+        "voting_period": "1210000s"
```

Comment:
@ankurdotb Should we increase this value?

Reply:
> @ankurdotb Should we increase this value?

Really depends on the validator set (who controls the vote). There are a number of configuration options that can be changed through governance and things like recovering an expired IBC client. Would you be willing to wait 2-3 weeks for something like that?

Reply:
Sorry, my bad.  I meant decreasing because the current situation in TestNet shows that 2 weeks is too long.